plugins/CoreHome/vue/src/EnrichedHeadline/EnrichedHeadline.vue

```diff
@@ -202,5 +193,17 @@ export default defineComponent({
       return Matomo.helper.htmlEntities(v);
     },
   },
+  computed: {
+    showRateFeature() {
+      // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+      return (window as any).translations.Feedback_SendFeedback;
```

Comment:
Works as intended :+1: 

But perhaps we can get rid of the lint disabling? There are two ways I know of that should work without changing other modules/files:

```
// piwik_translations is typed in CoreVue
return !!window.piwik_translations.Feedback_SendFeedback;

// the fallback translation should also work after importing the function
return translateOrDefault('Feedback_SendFeeback') !== 'Feedback_SendFeeback';
```

Reply:
Good idea, using the translation method to check for the translation rather than directly checking the window object seems like the best approach :+1: